.eslintrc

```diff
@@ -1,8 +1,15 @@
 ---
+env:
+  node: true
```

Comment:
Use `shared-node-browser`
as we'll be linting browser specific js too.

Reply:
Using `shared-node-browser` gives me errors:

```
.../gci-leaders/generate.js
  1:12  error  'require' is not defined  no-undef
  2:18  error  'require' is not defined  no-undef
  3:13  error  'require' is not defined  no-undef
  4:14  error  'require' is not defined  no-undef

.../gci-leaders/scrap.js
   1:15  error  'require' is not defined  no-undef
   2:12  error  'require' is not defined  no-undef
  25:17  error  'process' is not defined  no-undef
```

Is adding on `browser: true` beneath equivalent? That seems to work fine.

Reply:
Yea, I usually use `browser` and `node` 

But `shared-node-browser` is listed on https://eslint.org/docs/user-guide/configuring#specifying-environments :thinking:

Reply:
Weird, ok. I can look into that more in a while, but for now adding both seems fine 👍